src/services/vehicleService.ts

```
@@ -0,0 +1,37 @@
+    console.error(`Error fetching vehicle details for ${uuid}:`, error);
+    throw error;
+  }
+}
+
+export async function getVehicle(guid: string): Promise<vehicleDetails | undefined> {
+  try {
+    const rez = await axiosInstance.get(`/${SERVICE}/${guid}`)
+    return rez.data
+  }
+  catch (error) {
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)